### ISSUE TITLE
Make a footer

### ISSUE THREAD

**Issue by ACM-CIET** [ACM-CIET/ACM_Hacking-Wizards#7]
**Add footer for the web application with useful details and links like socials,etc**

**Comment by Palakiew** [comment]
I am interested in working on this issue

**Comment by ACM-CIET** [comment]
Hey @Palakiew go ahead and feel free to contact us for any help, and really sorry for belated reply!

**Comment by vansh-attri** [comment]
Hi, can you please assign me this issue?

**Comment by ACM-CIET** [comment]
> Hi, can you please assign me this issue?

Yes @vansh-attri but add that in footer component in react components not in index.html

**Comment by vansh-attri** [comment]
@ACM-CIET I have not learnt React yet so please unassign me.

**Comment by ACM-CIET** [comment]
> @ACM-CIET I have not learnt React yet so please unassign me.

No problem @vansh-attri we love your efforts 🙌🚀

**Comment by siddharthisrani** [comment]
Hii @ACM-CIET I am intrested in working this issue .

**Comment by Yashop181** [comment]
I will take on the task of creating a responsive and visually appealing footer for the project. I'll ensure it adapts well to different screen sizes and fits the overall design of our website.
